isolated-contexts.bs

```diff
@@ -44,6 +570,104 @@ If the [{{IsolatedContext}}] [=extended attribute=] appears on an
 [=interface mixin member=], or
 [=namespace member=],
 it indicates that the construct is [=exposed=]
-only within a [=isolated context=].
+only within an [=isolated context=].
 The [{{IsolatedContext}}] extended attribute must not be used
 on any other construct.
+
+The [{{IsolatedContext}}] extended attribute must
+[=takes no arguments|take no arguments=].
+
+If [{{IsolatedContext}}] appears on an [=overloaded=] [=operation=],
+then it must appear on all overloads.
+
+The [{{IsolatedContext}}] [=extended attribute=] must not be specified both on
```

Comment:
I could see "both on" referring to a list of 2 items, but what follows is a list of 3 items. Should this be perhaps:

```
... must not be specified on more than one of the following:
```